third_party/toolchains/BUILD

```diff
@@ -0,0 +1,46 @@
+# Copyright 2017 gRPC authors.
```

Comment:
Why does this file belong in the `third_party` directory?

Reply:
This is recommended by the Foundry team.